src/components/subscriptions/LicenseStatus.jsx

```diff
@@ -14,7 +15,7 @@ export default function LicenseStatus({ user }) {
         case 'assigned':
           return {
             iconClassName: 'fa-hourglass-half text-muted',
-            text: 'Pending',
+            text: <>Pending<br /><p>Since {user.pendingSince}</p></>,  /* eslint-disable-line */
```

Comment:
Perhaps this might be a bit cleaner, and would remove the need to any custom CSS:

```
case 'assigned':
  return {
    iconClassName: 'fa-hourglass-half text-muted',
    text: (
      <React.Fragment>
        <span>Pending</span>
        <span className="d-block text-muted">Since {user.pendingSince}</span>
      </React.Fragment>
    ),
  };
```